index.html

```diff
@@ -10,6 +10,20 @@
   <body>
     <div class="main">
       <header>
+        <div class="modal">
+          <img src="./Image/Enabled.svg" class="close" alt="closeTag">
+           <div class="modal-body">
+             <ul class="modal-menu">
+               <li><a href="#">Portoflio</a></li>
+               <hr />
+               <li><a href="#">About</a></li>
+               <hr />
+               <li><a href="#">Contact</a></li>
+               <hr />
+             </ul>
+           </div>
+         </div>
+         <div class="border1"></div>
```

Comment:
- [ ] Whenever we click on any menu item, we should be able to go to its corresponding section on the page. Currently yours is not behaving like that, would you address this in order to  improve User experience?


https://user-images.githubusercontent.com/55918152/140930478-767d0c75-37ef-40c0-98af-72f8a3bb1caf.mp4